mqtt-streaming/src/test/scala/docs/scaladsl/MqttFlowSpec.scala

```diff
@@ -98,15 +103,19 @@ class MqttFlowSpec
                 .toMat(BroadcastHub.sink)(Keep.both)
                 .run()
 
+              val subscribed = Promise[Done]
```

Comment:
This promise is about improving the reliability of the test i.e. we don't publish messages on a topic until we know that a client has subscribed.